# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glib/no-libelf.patch.cross
# Copyright (C) 2019 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- glib-2.60.6/gio/meson.build.vanilla	2019-09-11 21:51:12.296622092 +0000
+++ glib-2.60.6/gio/meson.build	2019-09-11 21:51:27.004621656 +0000
@@ -890,7 +890,8 @@
 endif
 
 if have_libelf
-  glib_conf.set('HAVE_LIBELF', 1)
+  #glib_conf.set('HAVE_LIBELF', 1)
+  libelf = []
 else
   libelf = []
 endif
